vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -850,15 +845,10 @@ func (e *StandardTxExecutor) SetSubnetValidatorWeightTx(tx *txs.SetSubnetValidat
 	if err != nil {
 		return err
 	}
-	if warpMessage.NetworkID != e.Ctx.NetworkID {
```

Comment:
Why are the checks for NetworkID removed?

Reply:
These checks are performed by the warp verification module now.